tests/vmi_networking_test.go

```diff
@@ -789,6 +789,91 @@ var _ = Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			table.Entry("without a specific port number", []v1.Port{}, defaultCIDR),
 			table.Entry("with custom CIDR", []v1.Port{}, customCIDR),
 		)
+
+		When("performing migration", func() {
+			var vmi *v1.VirtualMachineInstance
+			var ping func()
+
+			ping4 := func() {
+				pingVirtualMachine(vmi, gatewayIPFromCIDR(api.DefaultVMCIDR), "\\$ ")
```

Comment:
There is a need to use both the DG and an external address (e.g. 8.8.8.8).

Reply:
We should not test connectivity to remote IPs. That would prevent us from running these tests on disconnected environments.

And pinging to GW does not check the connectivity outside virt-launcher Pod.

We should do the same as in the tests above and create a client/server VMIs where we check connectivity within the cluster.

Reply:
I have change it to ping the virt-handler pod.